website/src/models/user.js

```diff
@@ -643,27 +648,22 @@ schema.methods.unlink = function (options, cb) {
 
   let self = this;
 
-  switch (keep) {
```

Comment:
The linter saw these as indentation errors.

In general, since none of these cases required us to fall through to another option, it's safer to use if/else if than a switch statement. Less likelihood of someone forgetting to put in a break statement.